server/migrations/002-user-array-support.ts

```diff
@@ -52,27 +52,27 @@ export async function up() {
     await model.save()
   }
 
-  const requests = await RequestModel.find({})
+  const approvals = await ApprovalModel.find({})
```

Comment:
This is a migration, so shouldn't be altered in this way.

Reply:
Yeah I wondered if we wanted to change it or not. It'll complain about `RequestModel` not existing anymore, but will revert any changes to this file.